tcf_website/templates/base/bugform.html

```diff
@@ -12,13 +12,17 @@ <h5 class="modal-title">Report a Bug </h5>
       </div>
       <div class="modal-body">
         <form class="needs-validation" id="bugform" novalidate>
-          <div class="form-row">
-              <label for="emailField">Email (leave blank to remain anonymous)</label>
-              <input type="email" class="form-control" id="emailField" placeholder="example@virginia.edu" value="">
-              <div class="invalid-feedback">
-                Invalid Email
-              </div>
-          </div>
+        {% if user.is_authenticated %}
+            <input type="hidden" id="emailField" value="{{user.email}}" readonly>
```

Comment:
Would it be better to just autofill and give them the option of deleting it? I don't imagine many people would care about the privacy concerns, though; my guess is that whoever isn't submitting their emails on reports is simply too lazy to type it in.

Reply:
Might be a little weird from a UX perspective to do this, though..?

Reply:
Also, _technically_ HTML can be tampered with by the user, so if we're going to use it anyway, there's no point in using hidden inputs. I'd rather use the user's email from the corresponding Django view, although I agree with @alex-shen1.

Reply:
Makes sense, changed to autofill